internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,138 @@
+package connect
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/pretty"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type OffsetConnectorOut struct {
+	Id       string `human:"ID" json:"id" yaml:"id"`
+	Name     string `human:"Name" json:"name" yaml:"name"`
+	Offsets  string `human:"Offsets" json:"offsets,omitempty" yaml:"offsets,omitempty"`
+	Metadata string `human:"Metadata" json:"metadata,omitempty" yaml:"metadata,omitempty"`
+}
```

Comment:
If this struct is only for human output, please get rid of the `json` and `yaml` tags since they are not used

Reply:
Done